### DISCUSSION TITLE
Way to access remote video

### DISCUSSION THREAD

**Discussion by tranzmatt** [vladmandic/human#398] [General]
Is there way to have a remote video stream (rtsp/udp/etc) used instead?  I have an IP camera overlooking my street I'd like to run this against.  I don't mind doing it myself if I know where to look.

**Comment by vladmandic** [comment]
majority of such video streams are not decodable by browser, but you may take a look at https://github.com/vladmandic/stream-rtsp and use that to create a local stream that you can then hook human into.
